### PR TITLE
[MNOE-399] Add ACL and settings

### PR DESCRIPTION
**Pull request by ouranos**

### CODE REVIEW

File: core/lib/generators/mno_enterprise/install/templates/config/settings.yml [hunk 1]
```diff
@@ -11,7 +11,9 @@ frontend_host:
 #===============================================
 # Feature Flags
 #===============================================
-
+# Display Audit Log in Organization Panel
```
Comment (fgourichon) [inline core/lib/generators/mno_enterprise/install/templates/config/settings.yml]: "Do not display Audit Log in..." ?

File: api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb [hunk 2]
```diff
@@ -17,6 +17,7 @@ angular.module('mnoEnterprise.configuration', [])
   .constant('PAYMENT_CONFIG', <%= Hash(Settings.payment).to_json %>)
   .constant('ORGANIZATION_MANAGEMENT', <%= Hash(Settings.organization_management).to_json %>)
   .constant('USER_MANAGEMENT', <%= Hash(Settings.user_management).to_json %>)
+  .constant('AUDIT_LOG', <%= Hash(Settings.user_management).to_json %>)
```
Comment (fgourichon) [inline api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb]: Isn't it 'admin_panel' ?

File: core/lib/generators/mno_enterprise/install/templates/config/settings.yml (hunk 1 above)
Comment (fgourichon) [inline core/lib/generators/mno_enterprise/install/templates/config/settings.yml]: "Do not display Audit log..." ?

File: api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb (hunk 2 above)
Comment (ouranos) [inline api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb]: My bad, to quick to copy/paste

**Comment by ouranos** [comment]
@fgourichon updated

**Pull request merged**